torch/_functorch/aot_autograd.py

```diff
@@ -2512,6 +2606,11 @@ def backward(ctx, *flat_args):
             all_args = (
                 list(ctx.symints) + list(ctx.saved_tensors) + list(contiguous_args)
             )
+
+            if config.functionalize_rng_ops:
+                # Add the seed and offset to args
+                seed, offset = CUDARngStateHelper.get_torch_state_as_tuple()
+                all_args = [seed, offset, *all_args]
```

Comment:
As this is hotpath, it's worth avoiding the second list reallocation here. I would compute `rng_args` and then concat it on 2607.